assets/src/styles/blocks/ENForm/components/_enform-side-style.scss

```diff
@@ -119,7 +119,7 @@
         padding-top: 0;
       }
 
-      .counter-block {
+      .counter-block _-- {
```

Comment:
In general I tried avoiding making spacing variable, but in this case it looked way off with our default spacing.